insights/parsers/aws_instance_id.py

```diff
@@ -5,8 +5,8 @@
 These parsers read the output of commands to collect identify information
 from AWS instances.
 
-* ``curl http://169.254.169.254/latest/meta-data/instance-identity/document`` and
-* ``curl http://169.254.169.254/latest/meta-data/instance-identity/pkcs7``
+* ``curl -s http://169.254.169.254/latest/meta-data/instance-identity/document`` and
+* ``curl -s http://169.254.169.254/latest/meta-data/instance-identity/pkcs7``
```

Comment:
I think the URL in this examples and the one below (line 24) should be `http://169.254.169.254/latest/dynamic/instance-identity/*`

Reply:
The specs are retrieving the specific items, `document` and `pkcs7`, so that each one can be parsed separately.

Reply:
Well, that I mean is that those 2 links are incorrect. It should be: 
* `curl -s http://169.254.169.254/latest/dynamic/instance-identity/document` and
* `curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7`

You can just compare those two to the actual simple_commands in `insights/specs/default.py`
Hopefully, that make sense?

Reply:
Yes, that makes sense, thanks for the feedback.